client/fingerprint/network.go

```diff
@@ -200,14 +199,57 @@ func (f *NetworkFingerprint) findInterface(deviceName string) (*net.Interface, e
 		return nil, err
```

Comment:
One of the comments on the PR indicated that the returned list here is not stable (can return things out of order). What happens if there is more than one interface with a global unicast address, thus resulting in them getting the same score? 
Reading the code below, it seems like it will choose the first one, but if the order changes when calling `interfaceDetector.Interfaces()` that will change the results.